### PR TITLE
Add live connector feature button to demo app

### PR DESCRIPTION
**Pull request by yuqil-amzn**
**Issue #:**
N/A
**Description of changes:**
This change adds support for media capture pipelines to the demo meeting applications.
**Testing:**
Deploy a demo app and verify
*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
Yes. 
```
npm run deploy -- -r us-east-1 -b test-lc-bucket2 -i us-east-1 -s yuqil-test1 --ivs-channel-prefix testButtonDemo123  -a meeting
```
Note:
This PR has the same content as [this PR](https://github.com/aws/amazon-chime-sdk-js/pull/2428), which has been approved. 

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
N/A

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
No

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Screenshot of the demo app:

Screen Shot 2022-09-12 at 13 19 07
Screen Shot 2022-09-12 at 13 21 39
Screen Shot 2022-09-14 at 15 26 23

### CODE REVIEW

**Comment by hensmi-amazon** [comment]
Think you need to run `npm run build:release` again locally. Also can you link https://github.com/aws/amazon-chime-sdk-js/pull/2428 in description and close that previous PR?

File: package.json [hunk 1]
```diff
@@ -101,6 +101,7 @@
     "@aws-sdk/client-chime-sdk-messaging": "^3.0.0",
     "@aws-sdk/util-hex-encoding": "^3.47.0",
     "@types/ua-parser-js": "^0.7.35",
+    "aws-sdk": "^2.1216.0",
```
Comment (devalevenkatesh) [inline package.json:104]: So for the demo changes we do not need `aws-sdk` dependency in the JS SDK root right?

File: package.json [hunk 2]
```diff
@@ -92,7 +92,7 @@
     "sinon": "^7.3.2",
     "spawn-wrap": "^2.0.0",
     "ts-node": "^9.1.1",
-    "typedoc": "^0.21.2",
+    "typedoc": "0.21.2",
```
Comment (devalevenkatesh) [inline package.json:95]: Any reason for this change?

Comment (yuqil-amzn) [reply package.json:95]: Sichao helped me resolve this. The reason is if not using the exact version, my change will build lots of case changes.

File: package.json (hunk 1 above)
Comment (yuqil-amzn) [reply package.json]: removed

File: demos/browser/README.md [hunk 3]
```diff
@@ -36,6 +36,14 @@ the same AWS region as the meeting and have the following bucket policy:
     ]
 }
 ```
+IVS channel is created in serverless and not in local. To use the Amazon Chime SDK media pipelines live connector feature, you need to provide an Amazon Interactive Video Service (IVS) channel endpoint and set the environment variable when running locally.
```
Comment (devalevenkatesh) [inline demos/browser/README.md:39]: nit: To use the Amazon Chime SDK media pipelines live connector feature, you need to provide an Amazon Interactive Video Service (IVS) channel endpoint and set the environment variable when running locally. IVS channel is created in only in the serverless browser demo and not in local browser demo.

**Review by devalevenkatesh — DISMISSED** [review]

**Review by devalevenkatesh — APPROVED** [review]

**Pull request merged**